cmd/keytransparency-server/frontend.go

```diff
@@ -208,6 +229,7 @@ func main() {
 	}
 	grpcServer := grpc.NewServer(opts...)
 	pb.RegisterKeyTransparencyServiceServer(grpcServer, svr)
+	trillian.RegisterTrillianLogServer(grpcServer, logServer)
```

Comment:
This PR looks good to me. I am just wondering what are pros and cons of running trillian in the same binary with Key Transparency as opposed to running it in its own binary. Wouldn't this line expose all trillian gRPC APIs on the same address and port of Key Transparency? Is that what we want to do?